jujugui/static/gui/src/app/components/header-help/header-help.js

```diff
@@ -58,7 +58,7 @@ class HeaderHelp extends React.Component {
   _generateIssuesLink() {
     let label = 'File Issue';
     let link = 'https://github.com/juju/juju-gui/issues';
-    if (this.props.gisf && this.props.user) {
+    if (this.props.user) {
```

Comment:
@frankban what if we created another landing page that was identical but with some of the wordings/links changed? But in the mean time updating the link to point to https://jujucharms.com/docs/stable/about-juju in the gisf case?